services/src/main/java/org/keycloak/services/resources/account/AccountConsole.java

```diff
@@ -126,13 +130,14 @@ public Response getMainPage() throws IOException, FreeMarkerException {
             }
             map.put("isTotpConfigured", isTotpConfigured);
 
+            map.put("deleteAccountAllowed", UserHelper.isDeleteAccountAllowed(realm, user));
```

Comment:
If we plan not to implement this feature in the old console, we should probably revert changes in this file.

Reply:
Sure ? this is service for the new console

AFAIK the old account console is handled by `AccountFormService`

Reply:
@zak905 Ah, I'm sorry. You are, of course, correct.